developer/system/gui.md

```diff
@@ -81,34 +91,34 @@ Certainly, it would be insecure to allow AppVM to read/write the clipboards of o
 
 This way, the user can quickly copy clipboards between AppVMs.
 This action is fully controlled by the user, it cannot be triggered/forced by any AppVM.
 
-*qubes_gui* and *qubes_guid* code notes
+`qubes-gui` and `qubes-guid` code notes
 -----------------------------------------
 
-Both applications are structured similarly. They use *select* function to wait for any of these two event sources:
+Both applications are structured similarly. They use `select` function to wait for any of these two event sources:
 
 - messages from the local X server
 - messages from the vchan connecting to the remote party
 
-The XEvents are handled by the *handle_xevent_eventname* function, and messages are handled by *handle_messagename* function. One should be very careful when altering the actual *select* loop, because both XEvents and vchan messages are buffered, and  *select* will not wake for each message.
+The XEvents are handled by the `handle_xevent_eventname` function, and messages are handled by `handle_messagename` function. One should be very careful when altering the actual `select` loop, because both XEvents and vchan messages are buffered, and  `select` will not wake for each message.
 
-If one changes the number/order/signature of messages, one should increase the *QUBES_GUID_PROTOCOL_VERSION* constant in *messages.h* include file.
+If one changes the number/order/signature of messages, one should increase the `QUBES_GUID_PROTOCOL_VERSION` constant in `messages.h` include file.
 
-*qubes_guid* writes debugging information to */var/log/qubes/qubes.domain_id.log* file; *qubes_gui* writes debugging information to */var/log/qubes/gui_agent.log*.
+`qubes-guid` writes debugging information to `/var/log/qubes/qubes.domain_id.log` file; `qubes-gui` writes debugging information to `/var/log/qubes/gui_agent.log`.
 Include these files when reporting a bug.
 
-AppVM -> dom0 messages
+AppVM -> GuiVM messages
 -----------------------
 
 Proper handling of the below messages is security-critical.
-Observe that beside two messages (`CLIPBOARD` and `MFNDUMP`) the rest have fixed size, so the parsing code can be small.
+Note that all messages except for `CLIPBOARD`, `MFNDUMP`, and `WINDOW_DUMP` have fixed size, so the parsing code can be small.
 
-The *override_redirect* window attribute is explained at [Override Redirect Flag](https://tronche.com/gui/x/xlib/window/attributes/override-redirect.html). The *transient_for* attribute is explained at [Transient_for attribute](https://tronche.com/gui/x/icccm/sec-4.html#WM_TRANSIENT_FOR).
+The `override_redirect` window attribute is explained at [Override Redirect Flag](https://tronche.com/gui/x/xlib/window/attributes/override-redirect.html). The `transient_for` attribute is explained at [Transient\_for attribute](https://tronche.com/gui/x/icccm/sec-4.html#WM_TRANSIENT_FOR).
```

Comment:
Is there really a need for the `\_` here? Can't it just be `[Transient_for attribute]` (i.e., a normal underscore not escaped)?